From 08cc0787968e57d36e4aa8f12b805d8ec9a28288 Mon Sep 17 00:00:00 2001
From: Gary Buhrmaster <gary.buhrmaster@gmail.com>
Date: Tue, 14 Jul 2020 15:08:04 +0000
Subject: [PATCH 5/7] use system videodev2.h include rather than a local
 (sometimes different) copy

The project no longer supports distributions with the older kernels that
required having a local copy of videodev2.h.
---
 mythtv/libs/libmythtv/recorders/v4lchannel.h | 2 +-
 mythtv/libs/libmythtv/v4l2util.h             | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/mythtv/libs/libmythtv/recorders/v4lchannel.h b/mythtv/libs/libmythtv/recorders/v4lchannel.h
index 365faa7ff1..d1064951dc 100644
--- a/mythtv/libs/libmythtv/recorders/v4lchannel.h
+++ b/mythtv/libs/libmythtv/recorders/v4lchannel.h
@@ -10,7 +10,7 @@
 #include "frequencies.h"
 
 #ifdef USING_V4L2
-#include "videodev2.h" // needed for v4l2_std_id type
+#include <linux/videodev2.h>
 #endif
 
 using namespace std;
diff --git a/mythtv/libs/libmythtv/v4l2util.h b/mythtv/libs/libmythtv/v4l2util.h
index 560931c1bd..955eca3a88 100644
--- a/mythtv/libs/libmythtv/v4l2util.h
+++ b/mythtv/libs/libmythtv/v4l2util.h
@@ -2,7 +2,7 @@
 #define V4L2_UTIL_H
 
 #ifdef USING_V4L2
-#include "videodev2.h" // our copy
+#include <linux/videodev2.h>
 #endif
 
 #include "tv.h"
-- 
2.26.2

